doc/Language/objects.pod6

```diff
@@ -447,10 +444,10 @@ After the C<BUILD> methods have been called, methods named C<TWEAK> are
 called, if they exist, again with all the named arguments that were passed
 to C<new>.
 
-Due to the nature of the default behavior of C<BUILDALL> and custom C<BUILD>
+Due to the default behavior of C<BUILDALL> and C<BUILD>
```

Comment:
The C< > were getting lost in the markdown, so added some ` to your comment.

How is this?
`Named arguments in a C<new> method call correspond directly to public attributes in dependent classes or named parameters in C<BUILD> or C<BUILDALL> submethods.  These are applied in in a Method Resolution Order (MRO).`